receipts/handler/handler.go

```diff
@@ -18,86 +17,45 @@ import (
 type ReceiptHandlerVersion int32
 
 const (
-	DefaultReceiptStorage  = 1 //ctypes.ReceiptStorage_CHAIN
-	ReceiptHandlerChain    = 1 //ctypes.ReceiptStorage_CHAIN
-	ReceiptHandlerLevelDb  = 2 //ctypes.ReceiptStorage_LEVELDB
-	ReceiptHandlerLegacyV1 = 101
-	ReceiptHandlerLegacyV2 = 102
-	DefaultMaxReceipts     = uint64(2000)
+	ReceiptHandlerLevelDb = 2 //ctypes.ReceiptStorage_LEVELDB
+	DefaultMaxReceipts    = uint64(2000)
 )
 
-func ReceiptHandlerVersionFromInt(v int32) (ReceiptHandlerVersion, error) {
-	if v < 0 || v > int32(ReceiptHandlerLegacyV2) {
-		return DefaultReceiptStorage, loomchain.ErrInvalidVersion
-	}
-	if v == 0 {
-		return ReceiptHandlerChain, nil
-	}
-	return ReceiptHandlerVersion(v), nil
-}
-
 // ReceiptHandler implements loomchain.ReadReceiptHandler, loomchain.WriteReceiptHandler, and
 // loomchain.ReceiptHandlerStore interfaces.
 type ReceiptHandler struct {
-	v               ReceiptHandlerVersion
 	eventHandler    loomchain.EventHandler
-	chainReceipts   *chain.StateDBReceipts
 	leveldbReceipts *leveldb.LevelDbReceipts
-
-	mutex         *sync.RWMutex
-	receiptsCache []*types.EvmTxReceipt
-	txHashList    [][]byte
-
-	currentReceipt *types.EvmTxReceipt
+	mutex           *sync.RWMutex
+	receiptsCache   []*types.EvmTxReceipt
+	txHashList      [][]byte
+	currentReceipt  *types.EvmTxReceipt
 }
 
 func NewReceiptHandler(
-	version ReceiptHandlerVersion, eventHandler loomchain.EventHandler,
+	eventHandler loomchain.EventHandler,
 	maxReceipts uint64, evmAuxStore *evmaux.EvmAuxStore,
-) (*ReceiptHandler, error) {
-	rh := &ReceiptHandler{
-		v:              version,
-		eventHandler:   eventHandler,
-		receiptsCache:  []*types.EvmTxReceipt{},
-		txHashList:     [][]byte{},
-		currentReceipt: nil,
-		mutex:          &sync.RWMutex{},
+) *ReceiptHandler {
+	return &ReceiptHandler{
+		eventHandler:    eventHandler,
+		receiptsCache:   []*types.EvmTxReceipt{},
+		txHashList:      [][]byte{},
+		currentReceipt:  nil,
+		mutex:           &sync.RWMutex{},
+		leveldbReceipts: leveldb.NewLevelDbReceipts(evmAuxStore, maxReceipts),
 	}
-
-	switch version {
-	case ReceiptHandlerChain:
-		rh.chainReceipts = &chain.StateDBReceipts{}
-	case ReceiptHandlerLevelDb:
-		leveldbHandler, err := leveldb.NewLevelDbReceipts(evmAuxStore, maxReceipts)
-		if err != nil {
-			return nil, errors.Wrap(err, "new leved db receipt handler")
-		}
-		rh.leveldbReceipts = leveldbHandler
-	}
-	return rh, nil
 }
 
 func (r *ReceiptHandler) Version() ReceiptHandlerVersion {
-	return r.v
+	return ReceiptHandlerLevelDb
```

Comment:
Is this used anywhere?

Reply:
Umm, look like it is an unused function, removed.